install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,67 +11,68 @@
-link:../../architecture/additional_concepts/storage.html[persistent storage] using
-https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS].
-Some familiarity with Kubernetes and NFS is assumed.
-
-The Kubernetes
-link:../../architecture/additional_concepts/storage.html[persistent volume]
-framework allows administrators to provision a cluster with persistent storage
-and gives users a way to request those resources without having any knowledge of
-the underlying infrastructure. Persistent volumes are not bound to a single
-project or namespace; they can be shared across the OpenShift cluster.
-link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
-volume claims], however, are specific to a project or namespace and can be
-requested by users.
-
-For a detailed example, see the guide for
-https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and
-MySQL using NFS].
 
-[IMPORTANT]
+OpenShift clusters can be provisioned with
+link:../../architecture/additional_concepts/storage.html[persistent storage]
+using NFS. Some familiarity with Openshift,
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent
+volume claims (PVCs)], and
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS]
+is beneficial.
+
+The Openshift persistent volume (PV) framework allows administrators to provision a cluster
+with persistent storage and gives developers the ability to request those resources without
+having specific knowledge of the underlying infrastructure. Persistent volumes are not bound
+to a single project; they can be shared across the entire OpenShift cluster. Persistent volume
+claims (PVCs), however, are specific to a project and are created and used by developers.
+
+[NOTE]
 ====
-High-availability of storage in the infrastructure is left to the underlying
-storage provider.
+_Project_ and _namespace_ are used interchangeably throughout this document. See
+link:../../architecture/core_concepts/projects_and_users.html#namespaces[Projects
+and Users] for details on the relationship.
 ====
 
 [[nfs-provisioning]]
-
 == Provisioning
-Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision NFS volumes in OpenShift, all that is
-required is a distinct list of NFS servers and paths and the
-`*PersistentVolume*` API.
 
-You must define your persistent volume in an object definition before creating
-it in OpenShift:
+Storage must exist in the underlying infrastructure before it can be mounted as
+a volume in OpenShift. To provision NFS volumes, a list of NFS servers and
+export paths are all that is required.
 
-.Persistent Volume Object Definition Using NFS
+[IMPORTANT]
 ====
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
```

Comment:
I dropped that sentence from the nfs doc and added it to the pod_security_context doc, since it is generic and applies to all of our supported storage.


----- Original Message -----
> >  ====
> > +High-availability of storage in the infrastructure is left to the
> > underlying
> > +storage provider.
> 
> @jeffvance I saw you removed this in your commit. I've added it back, but one
> section down. Did you feel strongly about dropping it?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/openshift-docs/pull/1726/files#r55769595